public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -800,24 +800,23 @@ block style-property-name-dart-diff
 :marked
   The component defines a `deleteRequest` property that returns an `EventEmitter`.
   When the user clicks *delete*, the component invokes the `delete()` method
-  which tells the `EventEmitter` to emit a `Hero` object.
+  telling the `EventEmitter` to emit a `Hero` object.
 
   Now imagine a hosting parent component that binds to the `HeroDetailComponent`'s `deleteRequest` event.
 
 +makeExample('template-syntax/ts/app/app.component.html',
 'event-binding-to-component')(format=".")
 :marked
   When the `deleteRequest` event fires, Angular calls the parent component's `deleteHero` method,
-  passing the *hero-to-delete* (emitted by `HeroDetail`) in the `$event` variable.
+  passing the *hero-to-delete* (emitted by `HeroDetail`) into the `$event` variable.
```

Comment:
It's the first one. The value of `$event` already _is_ the hero-to-delete (as set by the code `deleteRequest.emit(this.hero)`).